.idea/workspace.xml

```diff
@@ -18,7 +18,7 @@
         <entry file="file://$PROJECT_DIR$/reps_event_app/lib/ui/events.dart">
```

Comment:
Can you please explain why is the workspace.xml file being changed?

Reply:
I did not change anything in that file O.O

Reply:
You can check your commit log and go back to previous changes and see what caused `workspace.xml` to change.